apps/music/js/ui/views/player_view.js

```diff
@@ -274,6 +274,7 @@ var PlayerView = {
 
   setRepeat: function pv_setRepeat(value) {
     var repeatClasses = ['repeat-off', 'repeat-list', 'repeat-song'];
+    var repeatLabels = ['repeat-off', 'repeat-list', 'repeat-song'];
```

Comment:
I don't think we need this at the moment. We could just reuse `repeatClasses` (until we need to update strings, which may be never).